packages/components/pages/f-offers/src/assets/scss/common.scss

```diff
@@ -1 +1 @@
-@import '@justeat/fozzie/src/scss/fozzie';
+// Add styles here so it can be injected first via vue.config.js.
```

Comment:
Would a `@forward` work here so fozzie doesn't have to be imported in each component file?